tests/ui/async-await/track-caller/panic-track-caller.rs

```diff
@@ -81,7 +88,7 @@ async fn foo_closure() {
 
 // Since compilation is expected to fail for this fn when using
 // `nofeat`, we test that separately in `async-block.rs`
-#[cfg(feat)]
+#[cfg(cls)]
```

Comment:
Why is this using `cls` and not `afn`? This is an async block -- shouldn't it only work when `async_fn_track_caller` is enabled?

Reply:
Well, I guess it's separate.

Reply:
yeah it's separate, this case is still gated by `closure_track_caller`